media-plugins/vidstab/vidstab-1.1.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib
+
+MY_PN="vid.stab"
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+SRC_URI="https://github.com/georgmartius/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_sse2 openmp"
+PATCHES=( "${FILESDIR}/${PN}-gentoo.diff" )
+RDEPEND="openmp? ( sys-devel/gcc[openmp] )"
+DEPEND="${RDEPEND}"
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+src_configure() {
+        use openmp && append-cppflags "-fopenmp -DUSE_OMP"
+        use cpu_flags_x86_sse2 && append-cppflags "-DUSE_SSE2 -msse2 -ffast-math"
```

Comment:
this whole block needs to be rewritten:
```
if use openmp; then
	tc-check-openmp
	append-cppflags -DUSE_OMP
	append-cflags -fopenmp
fi
if use cpu_flags_x86_sse2; then
	append-cppflags -DUSE_SSE2
	append-cflags -msse2
fi
```
and yes, don't add `-ffast-math` to `CFLAGS`, that is best left to the user. Don't forget to inherit toolchain-funcs for `tc-check-openmp` and `flag-o-matic` for `append-*`